examples/psp22/lib.rs

```diff
@@ -31,6 +31,58 @@ pub mod my_psp22 {
         hated_account: AccountId,
     }
 
+    impl InternalImpl for Contract {}
+
+    impl Internal for Contract {
```

Comment:
I remember we were talking about creating macro for this thing, so overall that looks good

Reply:
yes sure